src/plots/wheelPlot.ts

```diff
@@ -0,0 +1,222 @@
+      if (endAngleAccessor != null) {
+        this._bindProperty(Wheel._END_ANGLE_KEY, endAngleAccessor, scale);
+      }
+
+      this._bindProperty(Wheel._START_ANGLE_KEY, startAngle, scale);
+      this.render();
+      return this;
+    }
+
+    /**
+     * Gets the AccessorScaleBinding for the end angle.
+     */
+    public endAngle<T>(): AccessorScaleBinding<T, number>;
+    /**
+     * Sets the end angle to a constant number or the result of an Accessor<number>.
+     *
+     * @param {number|Accessor<number>} endAngle
+     * @returns {Wheel} The calling Wheel Plot.
+     */
+    public endAngle(endAngle: number | Accessor<number>): Plots.Wheel<R, T>;
+    public endAngle<T>(endAngle?: number | Accessor<number> | T | Accessor<T>): any {
+      if (endAngle == null) {
+        return this._propertyBindings.get(Wheel._END_ANGLE_KEY);
+      }
+
+      let startAngleBinding = this.startAngle();
+      let angleScale = startAngleBinding && startAngleBinding.scale;
+      this._bindProperty(Wheel._END_ANGLE_KEY, endAngle, angleScale);
+
+      this.render();
+      return this;
+    }
+
+    /**
+     * Gets the AccessorScaleBinding for the inner radius.
+     */
+    public innerRadius<R>(): AccessorScaleBinding<R, number>;
+    /**
+     * Sets the inner radius to a constant number or the result of an Accessor<number>.
+     *
+     * @param {number|Accessor<number>} innerRadius
+     * @returns {Wheel} The calling Wheel Plot.
+     */
+    public innerRadius(innerRadius: number | Accessor<number>): Plots.Wheel<R, T>;
+    /**
+     * Sets the inner radius to a scaled constant value or scaled result of an Accessor.
+     * The provided Scale will account for the values when autoDomain()-ing.
+     *
+     * @param {R|Accessor<R>} innerRadius
+     * @param {Scale<R, number>} scale
+     * @returns {Wheel} The calling Wheel Plot.
+     */
+    public innerRadius<R>(innerRadius: R | Accessor<R>, scale: Scale<R, number>): Plots.Wheel<R, T>;
+    public innerRadius<R>(innerRadius?: number | Accessor<number> | R | Accessor<R>, scale?: Scale<R, number>): any {
+      if (innerRadius == null) {
+        return this._propertyBindings.get(Wheel._INNER_RADIUS_KEY);
+      }
+
+      if (scale != null) {
+        scale.range([0, Math.PI * 2]);
+      }
+
+      let outerRadiusBinding = this.outerRadius();
+      let outerRadiusAccessor = outerRadiusBinding && outerRadiusBinding.accessor;
+      if (outerRadiusAccessor != null) {
+        this._bindProperty(Wheel._OUTER_RADIUS_KEY, outerRadiusAccessor, scale);
+      }
+
+      this._bindProperty(Wheel._INNER_RADIUS_KEY, innerRadius, scale);
+      this.render();
+      return this;
+    }
+
+    /**
+     * Gets the AccessorScaleBinding for the outer radius.
+     */
+    public outerRadius<R>(): AccessorScaleBinding<R, number>;
+    /**
+     * Sets the outer radius to a constant number or the result of an Accessor<number>.
+     *
```

Comment:
Can we add some documentation explaining that it uses the same `Scale` as `innerRadius()`?